public/js/app.js

```diff
@@ -0,0 +1,210 @@
+'use strict';
+
+let navigationButtons = document.getElementsByClassName('navButtons');
+for (let i = 0; i < navigationButtons.length; i++) {
+  navigationButtons[i].addEventListener('click', swapSelected);
+}
+
+function swapSelected() {
+  let selected = this.innerHTML;
+  const contentBox = document.getElementById('content');
+
+  if (selected === 'r/JavaScript') {
+    contentBox.innerHTML = '';
+    const rSlashJavaScript = new XMLHttpRequest();
+    rSlashJavaScript.addEventListener('load', reqListenerJavaScript);
+    rSlashJavaScript.open('GET', 'https://www.reddit.com/r/javascript/.json');
+    rSlashJavaScript.send();
+
+    function reqListenerJavaScript() {
```

Comment:
This function is the same across all your ifs. This means it could be pulled out into a function and be in one place rather than copied across 4 if conditionals.